partiql-planner/build.gradle.kts

```diff
@@ -112,19 +112,46 @@ tasks.register<Copy>("copyUtils") {
 //
 // !! IMPORTANT !! — only run manually, as this will overwrite the existing ir/Nodes.kt.
 //
-tasks.register<Copy>("copyNodes") {
-    includeEmptyDirs = false
-    dependsOn("codegen")
-    filter { it.replace(Regex("public (?!(override|(fun visit)))"), "internal ") }
-    from("$buildDir/tmp")
-    include("**/Nodes.kt")
-    into("src/main/kotlin")
-}
+// tasks.register<Copy>("copyNodes") {
+//     includeEmptyDirs = false
+//     dependsOn("codegen")
+//     filter { it.replace(Regex("public (?!(override|(fun visit)))"), "internal ") }
+//     from("$buildDir/tmp")
+//     include("**/Nodes.kt")
+//     into("src/main/kotlin")
+// }
 
 tasks.register("generate") {
     dependsOn("codegen", "copyUtils")
 }
 
 tasks.compileKotlin {
     dependsOn("generate")
+    dependsOn(tasks.withType<Copy>())
```

Comment:
These additions to other packages (i.e. everything not `lang` and `planner`) were added following the upgrade to gradle 8+. In our previous gradle version, 7+, there were some task dependencies that weren't deterministic, which became errors in gradle 8+. So these additional `dependsOn` blocks were needed for this PR.

The shadowjar plugin is added to all of the published packages to keep the publishing configuration the same for all packages. The published artifacts for the other packages (not `lang` or `parser`) will be the same since there's no `shadowJar` configuration in their gradle build files.